frontend/src/FullPageRoutes/FAQ.tsx

```diff
@@ -0,0 +1,169 @@
+
+        <Grid item>
+          <Grid
+            container
+            justify="center"
+            alignItems="center"
+            spacing={1}
+            className={classes.bottomBar}
+          >
+            <Grid item>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>How can I sign up for Conteo?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    To sign up, go to our homepage and select the sign up
+                    options. You would then be asked to provide your email and
+                    password.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>Who are penpals?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    Penpals are anyone you are matched to.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>
+                    How many video message can be sent per day?
+                  </Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    One (1) video messages be per day can be sent.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>How long can the video be?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    The video can be between fifteen seconds to 10 minutes long.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+            </Grid>
+
+            <Grid item>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>What are the privacy settings?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    The privacy settings are public, private and hidden. Public
```

Comment:
"The privacy settings let you control what information is shared with other users.
Public allows anyone that you match with to see that information. For instance, if you set your age to public, anyone you match with will see your age.
Private allows only your penpals to see the information. If you set your age to private, only your penpals can see your age.
Hidden means no one but you can see the information. 
If you set something to hidden, that means it can't be used to match you with other users. For instance, if you are 33, and you set your age to hidden, another user looking for penpals who are 30-40 years old will not be matched to you."